src/styles/card.scss

```diff
@@ -0,0 +1,21 @@
+.card {
+  display: flex;
+  flex-direction: column;
+
+  &__name {
+    padding-top: 207px;
+    font-weight: 500;
+    font-size: 12px;
+    color: $text-color-strong;
+    padding-inline: 16px;
+    transition: color $anim-duration ease;
+  }
+
+  &__product-code {
+    font-weight: 400;
+    font-size: 10px;
+    margin-top: 4px;
+    color: $text-color;
+    padding-left: $padding-left;
```

Comment:
The variable `$padding-left` is used but not defined. Please define this variable to avoid compilation errors.